osquery/CMakeLists.txt

```diff
@@ -235,7 +249,7 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   set_target_properties(libosquery_testing PROPERTIES OUTPUT_NAME osquery_testing)
 
   # osquery core set of unit tests build with SDK.
-  add_executable(osquery_tests main/tests.cpp ${OSQUERY_TESTS})
+  add_executable(osquery_tests main/tests_core.cpp ${OSQUERY_TESTS})
```

Comment:
`tests_core.cpp` modifies `main` to support `launchWorker` and `launchExtension` testing.